### PR TITLE
Add customer specific migration path

### PR DESCRIPTION
**Pull request by diamonwiggins**
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: testgrid/specs/full.yaml [hunk 1]
```diff
@@ -2661,3 +2661,58 @@
...
+    kotsadm: 
+      version: "1.101.x"
+    ekco: 
+      version: "0.27.x"
+    minio: 
+      version: "2023-06-29T05-12-28Z"
+    metricsServer: 
+      version: "0.6.x"
+    certManager:
+      version: "1.9.x"
```
Comment (laverya) [inline testgrid/specs/full.yaml:2718]: this spec does not include openebs

Comment (diamonwiggins) [reply testgrid/specs/full.yaml:2718]: it sure doesn't, whoops

**Comment by diamonwiggins** [comment]
@laverya sorry hadn't had a chance to get back this to this one yet. Thanks for the updates. Anything else you need from me on this one?

**Comment by diamonwiggins** [comment]
Putting this one in draft for now as we finalize what the customer's exact migration path will be.

**Comment by laverya** [comment]
I'm going to give this one another test, my openebs update last Friday might have made it work

**Comment by laverya** [comment]
Well, that and "the Longhorn to OpenEBS migration used to exit early if k8s was being upgraded past 1.23.x too" 😄

**Pull request closed without merging**